### DISCUSSION TITLE
Hide topbar on fullscreen mode (f11)

### DISCUSSION THREAD

**Discussion by redactedontop** [zen-browser/desktop#3131] [Ideas]
Hiya!
I'd love a toggle to do this, like other browsers do.

**Comment by yZibi** [comment]
Add 

> `#main-window[sizemode="fullscreen"] #nav-bar {margin-top: -40px;}`

 to  `userChrome.css`

 located in `%appdata%\zen\profiles\ @yourprofile@\chrome\`

**Comment by TheTrueWhiteOwl** [comment]
I would place the following as a toggle in the settings:
Enable Compact Mode on Fullscreen

See also discussion #801
